### ISSUE TITLE
Bounds and Locatable problem

### ISSUE THREAD

**Issue by JanKoehnlein** [theia-ide/sprotty#67]
I'd expect the bounds of a `ViewportRoot` to be the combined bounds of its children.

Currently, we pretend a `Viewport` would have bounds, but in fact the bounds never contain something useful. If we e.g. set the position, the new position gets applied as a transform on the SVG element, translating the entire canvas on the page. This is definitely not what I expected.

The problem is that we add the transform to the SVG element when its 
`SModelElement` is `Locatable`, i.e. has a `position` property. So my first idea was to replace the `position` and `dimension` property of the `ViewportRoot` by a `bounds` property. Interestingly, this didn't work, as the `position` property is set by the `ModelSource` when applying the bounds, as it cannot tell from the schema whether to use `position` and `dimension` or just `bounds`.

I cannot really think of a way to implement that that's neither magic or inconvenient.
1) Don't add the SVG transform based on `Locatable`.
   Do it manually in each View (inconvenient, makes pre-rendered elements difficult), or use `Moveable` again (strangely asymmetric), or make an exception for ViewportRoot (magic, dependency wrong way) .
2) Make a special case in the `SModelFactory` (magic)
3) Make a special case in the `ModelSource` (magic, but the bounds->position & dimension mapping there is magic anyway)
4) Use a different property (inconvenient, causes strange asymmetries in client code)

Summing up, we now have three types of elements with a position
1) moveable ones (transform in SVG)
2) locatable ones (transform in SVG)
3) the viewport (no transform in SVG)
to handle differently.
